content-repo/extra-docs/articles/microsoft-auth-guide.md

```diff
@@ -64,9 +64,13 @@ For example, for Microsoft Graph User, replace the ***SCOPE*** with `directory.a
 9. Enter your redirect URI in the ***Redirect URI*** parameter field.
 
 
-## Revoke Consent
+## Revoking Consent
+In case you want to revoke the consent given to a usage of an application(on behalf of a user or an admin consent), please follow the instructions below:
+
+1. In order to revoke consent to a Cortex XSOAR Microsoft application, refer to the [Microsoft documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-convert-app-to-be-multi-tenant#revoking-consent).
```

Comment:
need to work on the text here, you are writing basically twice "to revoke consent..."